domains/transfer/Transfer.API/API/Transfer/Api/Dto/Requests/CreateTransferAgreementProposal.cs

```diff
@@ -1,42 +1,41 @@
-using System;
-using API.Shared.Extensions;
 using FluentValidation;
 using Microsoft.AspNetCore.Http;
+using System;
+using API.Shared.Extensions;
 
 namespace API.Transfer.Api.Dto.Requests;
 
-public class CreateTransferAgreementValidator : AbstractValidator<CreateTransferAgreement>
+public record CreateTransferAgreementProposal(long StartDate,
+    long? EndDate,
+    string ReceiverTin);
```

Comment:
According to the UX, shouldnt the receiverTin be optional?

Reply:
I can't tell. I asked Rene

Reply:
Although a TA needs a ReceiverTin

Reply:
ReceiverTin is now optional for proposals